src/harness/harness.ts

```diff
@@ -1566,21 +1566,32 @@ module Harness {
         export interface BaselineOptions {
             LineEndingSensitive?: boolean;
             Subfolder?: string;
+            Baselinefolder?: string;
         }
 
-        export function localPath(fileName: string, subfolder?: string) {
-            return baselinePath(fileName, 'local', subfolder);
+        export function localPath(fileName: string, baselinefolder?: string, subfolder?: string) {
+            if (baselinefolder === undefined) {
+                return baselinePath(fileName, 'local', 'tests/baselines', subfolder);

```

Comment:
Why not just always take the baselineFolder and just have each runner pass in the right thing?